docs: update readme for a380x (#8366)
flybywiresim/aircraft: README.md

![FlyByWire Simulations](https://raw.githubusercontent.com/flybywiresim/branding/1391fc003d8b5d439d01ad86e2778ae0bfc8b682/tails-with-text/FBW-Color-Light.svg#gh-dark-mode-only)
![FlyByWire Simulations](https://github.com/flybywiresim/branding/blob/master/tails-with-text/FBW-Color-Dark.svg#gh-light-mode-only)

# FlyByWire Simulations Aircraft

[![GitHub latest release version](https://img.shields.io/github/v/release/flybywiresim/aircraft.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![Github All Releases download count](https://img.shields.io/github/downloads/flybywiresim/aircraft/total.svg?style=flat)](https://github.com/flybywiresim/a32nx/releases/latest)
[![GitHub contributors](https://img.shields.io/github/contributors/flybywiresim/aircraft.svg?style=flat)](https://github.com/flybywiresim/a32nx/graphs/contributors)

> [![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/UjzuHMU)

Reviewer: ```suggestion
[![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/flybywire)

```

> [![Twitter](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=twitter&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://twitter.com/FlybywireSim)

Reviewer: ```suggestion
[![Twitter](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=X&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://x.com/FlybywireSim)

```

[![YouTube](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=youtube&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.youtube.com/c/FlyByWire-Simulations)
[![Facebook](https://img.shields.io/badge/-FlyByWireSimulations-e84393?label=&logo=facebook&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://www.facebook.com/FlyByWireSimulations/)
[![Instagram](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=instagram&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://instagram.com/flybywiresim)

> 

Reviewer: ```suggestion
[![Bluesky](https://img.shields.io/badge/-@FlyByWireSim-e84393?label=&logo=Bluesky&logoColor=ffffff&color=6399AE&labelColor=00C2CB)](https://bsky.app/profile/flybywiresim.com)

```

> The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 n Microsoft Flight Simulator that are as close to reality as possible.

Reviewer 2: ```suggestion
The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.
```

> The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.

Reviewer 3: ```suggestion
The A32NX and A380X projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.
```

Reviewer 4: Might as well fix the weird phrasing too, and also correct the tense.
```No, they are completely free and open source, along with SimBridge and other FlyByWire projects which are publicly accessible via GitHub.```

> The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.

Reviewer 4: For consistency
```suggestion
The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320-200N and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.
```
or
```
The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320neo and A380 in Microsoft Flight Simulator that are as close to reality as possible.
```

> The A32NX and A380X Projects are community-driven open source projects to recreate, respectively, a free Airbus A320-200N and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.

Reviewer 3: is "respectively" important?

```suggestion
The A32NX and A380X Projects are community-driven open source projects to recreate a free Airbus A320-200N and A380-800 in Microsoft Flight Simulator that are as close to reality as possible.
```

 WXR         Honeywell RDR-4000
 ```

### A380-800 (Trent 900)

```
Model           A380-842
Engine          Rolls-Royce Trent 972B-84
Weight Variant  WV03
```

Please note that this configuration may change in the future as the projects evolve and change.

> > [!NOTE]  
> > While code and some assets for the A380X are present in this repository, no usable sim model is present yet as the aircraft is still in pre-alpha. You cannot compile this repository to produce a usable aircraft.

Reviewer 5: ```suggestion
> [!NOTE]  
> While code and some assets for the A380X are present in this repository, no usable sim model is present yet as the aircraft is still in pre-alpha. You cannot compile this repository to produce a usable aircraft.
```

This is now outdated

Author: Fixed

### What liveries are available?

> Liveries for the A32NX can be found on [Flightsim.to](https://flightsim.to/c/liveries/flybywire-a32nx/). They will be available for the A380X once it is released in alpha.

Reviewer 3: ```suggestion
Liveries for the A32NX can be found on [Flightsim.to](https://flightsim.to/c/liveries/flybywire-a32nx/). 

Liveries for the A380X will be available once it is released in alpha and livery builders had some time to create them. 
```

### What liveries are available?

Liveries for the A32NX can be found on [Flightsim.to](https://flightsim.to/c/liveries/flybywire-a32nx/). 

> Liveries for the A380X will be available once it is released in alpha and livery builders had some time to create them. 

Reviewer: ```suggestion
Liveries for the A380X can be found on [Flightsim.to](https://flightsim.to/liveries/flybywire-a380x/).

```

Read [Contributing.md](.github/Contributing.md) and join our Discord to get started.

> ### Are the aircraft payware?

Reviewer 3: ```suggestion
### Are FlyByWire aircraft payware?

```

> No, they are completely free aircraft, open-source.

Reviewer 3: ```suggestion
No, they are completely free aircraft, open-source, including SimBridge and other FlyByWire projects which are publicly accessible via GitHub.
```

This way we do not block any future developments/ideas.

Reviewer 4: Might as well fix the weird phrasing too:
```
No, they are completely free and open source, along with SimBridge and other FlyByWire projects that are publicly accessible via GitHub.
```

### How do we report bugs?

Please read the [known Issues and bug reporting](#known-issues-and-bug-reporting) section.

### Is this going to be "study-level"?

> While many in the team dislike the term "study-level" as its use is mostly applicable to certified training devices, we are in fact very dedicated on bringing a high fidelity A320neo and A380-800 to the Microsoft Flight Simulator platform.

Reviewer 4: As above, not consistent.
